verbit/streaming_client.py

```diff
@@ -528,7 +543,7 @@ def _handle_socket_close(self, data):
             if code not in (STATUS_NORMAL, STATUS_GOING_AWAY):
                 self._logger.warning('Unexpected close code: ' + msg)
             else:
-                self._logger.debug(msg)
+                self._logger.info(msg)
 
```

Comment:
@EliAsor In any case, don't we need to close the web socket here too, as in `_close_ws`?

Reply:
As we discussed f2f, the underlying websocket client will have already sent the "close" reply message to the server by this point, so we don't need to do anything else to close the connection here.